CardanoSharp.Blockfrost.Sdk/IAddressesClient.cs

```diff
@@ -7,4 +7,7 @@ public interface IAddressesClient
 {
 	[Get("/addresses/{address}/utxos?count={count}&page={page}&order={order}")]
 	Task<ApiResponse<AddressUtxo[]>> GetAddressUtxosAsync(string address, int count = 100, int page = 1, string order = "asc");
+
+    [Get("/addresses/{address}/transactions?count={count}&page={page}&order={order}&from={from}&to={to}")]
+    Task<ApiResponse<AssetTransaction[]>> GetTransactionsAsync(string address, string from, string to ,int count = 100, int page = 1, string order = "asc");
```

Comment:
nitpick, but should we rather put the from and to at the end and give them a default of blank?

Reply:
am a bit either way on the ordering, but it would be more consistent with the docs and with the other methods to have the count and page etc first.
do the from and to work with blank strings as defaults?